src/twisted/test/test_proto_helpers.py

```diff
@@ -0,0 +1,238 @@
+# Copyright (c) Twisted Matrix Laboratories.
```

Comment:
nitpick on file naming: my personal preference here would be to move all the tests here into `src/twisted/internet/test/test_testing.py` so there's only one place to look.

Reply:
Removed twisted.test.test_proto_helpers, and moved all deprecation tests to twisted.internet.test.test_testing.